MobileCenter/MobileCenter/Internals/Storage/MSFileUtil.m

```diff
@@ -69,7 +69,7 @@ + (BOOL)deleteFile:(MSFile *)file {
     MSLogVerbose([MSMobileCenter logTag], @"File %@: has been successfully deleted", file.fileURL);
     return YES;
   } else {
-    MSLogError([MSMobileCenter logTag], @"Error deleting file %@: %@", file.fileURL, error.localizedDescription);
+    MSLogError([MSMobileCenter logTag], @"Couldn't delete a file \"%@\": %@", file.fileURL, error.localizedDescription);
```

Comment:
@TroubleMakerBen, I agree with @clpolet, it should be consistent with other formulations. A few examples other messages:
- `Couldn't get files in the directory ...`
- `Couldn't create directory ...`
- `Couldn't load file ...`
- `Couldn't create regular expression ...`
- `Couldn't serialize log container ...`
- `Couldn't parse crash report ...`